pkg/docker/config/config.go

```diff
@@ -204,7 +208,17 @@ func GetAllCredentials(sys *types.SystemContext) (map[string]types.DockerAuthCon
 // The homeDir parameter should always be homedir.Get(), and is only intended to be overridden
 // by tests.
 func getAuthFilePaths(sys *types.SystemContext, homeDir string) []authPath {
```

Comment:
Adding the new paths to this function means that writers would never write to `systemPath` without a specific override; is that intended?

I think that’s actually a good idea, so that `root`’s interactive login doesn’t write to a noninteractively-used file, but it should be documented.